bakerydemo/jinja2/base/form_page.html

```diff
@@ -12,22 +11,20 @@ <h1 class="index-header__title">{{ page.title }}</h1>
                 {% if page.intro %}
                     <p class="intro">{{ page.intro|richtext }}</p>
                 {% endif %}
-                {% if page.body %}
-                    {{ page.body }}
-                {% endif %}
+                {{ page.body if page.body }}
```

Comment:
Hmm, this felt like a clever shortcut while writing this but maybe it’s better to stick with the previous syntax if we were to keep this Jinja version in sync.